### PR TITLE
connection issue solved

### PR DESCRIPTION
**Pull request by jijeesh**
there was a problem while connecting to active auto discovery. that is solved in the agent file

### CODE REVIEW

**Comment by jangaraj** [comment]
I'm confused. You are mixing Docker images:
- `monitoringartist/zabbix-xxl:latest` - obsolete image - the better one is `monitoringartist/dockbix-xxl:latest`
- `zabbix/zabbix-agent:latest` is not dockbix agent

`pv.yaml` is an empty file.

Your PR needs some cleaning + description what is trying to solve.

**Comment by jijeesh** [comment]
There is a problem to auto registration  to the zabbix server by active method.
The problem was the env value required double quote. so I added that changes in the file  dockbix-agent-xxl-daemonset.yaml
selection 

env:
            - name: ZA_Server
              value: "zabbix-server"
            - name: ZA_ServerActive
              value: "zabbix-server"

after this change only the agent is connect to the zabbix server via auto registration

File: dockbix-agent-xxl-daemonset.yaml [hunk 1]
```diff
@@ -19,18 +19,20 @@ spec:
...
-        - resources:
-            requests:
-              cpu: 0.15
+        - image: monitoringartist/dockbix-agent-xxl-limited
+          name: dockbix
           securityContext:
-            privileged: true
+             privileged: true
+          resources:
+            requests:k
```
Comment (jangaraj) [inline dockbix-agent-xxl-daemonset.yaml]: Is it correct syntax `requests:k`?

**Comment by jangaraj** [comment]
Thx, it is better. Why do you want to create ingress resource?

File: dockbix-agent-xxl-daemonset.yaml (hunk 1 above)
Comment (jijeesh) [reply dockbix-agent-xxl-daemonset.yaml]: k is wrong thing happen while saving the file, i have reset

**Comment by jijeesh** [comment]
> Thx, it is better. Why do you want to create ingress resource?

I added ingress if some one want to add domain for accessing the zabbix url , he can update in the host location. also update with nodePort if some one want to look directly through the node ip + port.

**Comment by jangaraj** [comment]
Thx.

**Pull request merged**